### ISSUE TITLE
Product taxonomy labels in admin screens

### ISSUE THREAD

**Issue by lutrov** [woothemes/woocommerce#11297]
There's an inconsistency in the admin screens, in terms of product taxonomy labels. Product categories use  "Product Categories" and "Add New Product Category". Product tags use "Product Tags" and "Add New Product Tag" but product attributes use "Attributes" and "Add New Attribute". This applies to H1 & H2 headers, as well as buttons.

**Comment by mikejolley** [comment]
What am I missing here? Are you talking about the word 'products'?

**Comment by lutrov** [comment]
Yes, specificall that the attributes screens doesn't mention the word but the other two taxonomy screens do.

**Comment by mikejolley** [comment]
How is this a problem then? To add reasoning, categories and tags and taxonomies used in POSTS. So there needs to be a difference between POST category and PRODUCT category.

Attributes on the otherhand are product only. No need to prefix.

**Comment by lutrov** [comment]
No need to prefix provided Woocommerce is the only plugin. Is this always true? No, it's not. I'm not suggesting it's  __problem__, it's an __inconsistency__ which has the potential to cause confusion. Hence, there __is__ a need to prefix.

**Comment by mikejolley** [comment]
![imgres](https://cloud.githubusercontent.com/assets/90977/16481121/761368d2-3ea1-11e6-8890-36b4c936c369.jpg) :)

I don't agree. WordPress doesn't prefix it's own taxonomy names, so why should WooCommerce?

@jameskoster see above - thoughts?

**Comment by claudiosmweb** [comment]
I think we should remove the "Product" from "Add New Product Category".

**Comment by lutrov** [comment]
Wordpress doesn't prefix it's own taxonomy names, so why should Woocommerce? Aren't you now contradicting yourself?

From what I can see, Woocommerce __does__ prefix taxonomies, except, it __doesn't do it consistently__. If we agree on the "why should Woocommerce", then we should be using "Categories" and "Add New Category",  "Tags" and "Add New Tag" instead.

**Comment by mikejolley** [comment]
As I said above, because with those 2 taxonomies, there are also **same-name taxonomies** (but different) for posts.

post_category vs product_category
post_tag vs product_tag

**Comment by lutrov** [comment]
![761368d2-3ea1-11e6-8890-36b4c936c369](https://cloud.githubusercontent.com/assets/11863829/16481435/9a28bfe0-3eee-11e6-9f8d-198a68675d0a.jpg)

:-)

**Comment by lutrov** [comment]
I don't understand your answer. Nevermind, it looks like it's a bit too hard.

**Comment by lutrov** [comment]
You'll notice the submenu's don't say "Product Categories" & "Product Tags". If we use your example:

> post_category vs product_category
> post_tag vs product_tag

The standard WP categories & tags are labelled "Categories" and "Tags" in the submenu. So why are the Woocommerce product submenu categories & tags also labelled as "Categories" and "Tags" then? See the problem?

**Comment by mikejolley** [comment]
Then I'd rather remove the prefix as @claudiosmweb said :)

**Comment by lutrov** [comment]
Removing the word "product" from "Product Categories" and "Add New Product Category", "Product Tags" and "Add New Product Tag" would be better than it is now.

Personally, I don't particularly care either way, as long there's consistency.

**Comment by claudiosmweb** [comment]
> Personally, I don't particularly care either way, as long there's consistency.

Ok :)
